Google.Api.Gax/UnparsedResourceName.cs

```diff
@@ -12,30 +12,30 @@ namespace Google.Api.Gax
     /// <summary>
     /// A resource name in which nothing is known about the name structure.
     /// </summary>
-    public sealed class UnknownResourceName : IResourceName, IEquatable<UnknownResourceName>
+    public sealed class UnparsedResourceName : IResourceName, IEquatable<UnparsedResourceName>
     {
         /// <summary>
-        /// Parse a resource name into an <see cref="UnknownResourceName"/>.
+        /// Parse a resource name into an <see cref="UnparsedResourceName"/>.
         /// Only minimal verification is carried out that <paramref name="name"/> is a value resource name string.
```

Comment:
Nit: value => valid, while we're editing this file?

Reply:
done